ui/segmented-bar.info/main.reel/main.html

```diff
@@ -0,0 +1,180 @@
+                "prototype": "ui/segmented-bar.reel",
+                "properties": {
+                    "element": {"#": "segmentedBar3"},
+                    "childrenPath": "regions",
+                    "contentController": {"@": "treeController"}
+                }
+            },
+            "treeController": {
+                "prototype": "montage/core/tree-controller",
+                "bindings": {
+                    "content": {"<-": "@owner.tree"}
+                }
+            },
+            "segment3": {
+                "prototype": "ui/segment.reel",
+                "properties": {
+                    "element": {"#": "segment3"}
+                },
+                "bindings": {
+                    "content": {
+                        "<-": "@segmentedBar3:segment.object.content.{label: region, length: size}"
+                    }
+                }
+            },
+
+
+            "segmentedBar4": {
+                "prototype": "ui/segmented-bar.reel",
+                "properties": {
+                    "element": {"#": "segmentedBar4"},
+                    "contentController": {"@": "rangeController"},
+                    "orientation": "vertical"
+                }
+            },
+            "rangeController": {
+                "prototype": "montage/core/range-controller",
+                "bindings": {
+                    "content": {"<-": "@owner.content"}
+                }
+            },
+            "segment4": {
+                "prototype": "ui/segment.reel",
+                "properties": {
+                    "element": {"#": "segment4"}
+                },
+                "bindings": {
+                    "content": {
+                        "<-": "@segmentedBar4:segment.object.{label: region, length: size}"
+                    }
+                }
+            }
+        }
+    </script>
+</head>
+
+<body>
+
+<div data-montage-id="main" class="Main">
+    <h1>Data from event listener</h1>
+    <pre data-montage-id="text"></pre>
+
+    <h1>Desktop - content array, isSelectionEnabled = true</h1>
+    <div class="desktop">
+        <div data-montage-id="segmentedBar1">
+            <div data-montage-id="segment1"></div>
+        </div>
+    </div>
+    <br/>
+
+    <h1>Desktop - RangeController + bootstrap</h1>
+    <div class="desktop">
+        <div data-montage-id="segmentedBar2" class="progress">
+            <div data-montage-id="segment2" class="progress-bar progress-bar-striped"></div>
+        </div>
+    </div>
+    <br/>
+
+    <h1>Desktop - TreeController</h1>
+    <div class="desktop">
+        <div data-montage-id="segmentedBar3">
+            <div data-montage-id="segment3"></div>
+        </div>
+    </div>
+    <br/>
+
+    <h1>Phone-portrait, orientation = vertical</h1>
+    <div class="phone-portrait">
+        <div data-montage-id="segmentedBar4">
+            <div data-montage-id="segment4"></div>
+        </div>
+    </div>
+</div>
+
+</body>
+</html>
```

Comment:
Can we have an example (or all examples) with differently colored segments? This will probably be the usual use case for segmented bar, so we should show how it's done. This will also provide us with verification that it's as easy to do as it should be.